code/scripting/api/objs/graphics.cpp

```diff
@@ -164,18 +164,23 @@ ADE_FUNC(__len, l_Graphics_Posteffects, NULL, "Gets the number or available post
 	return ade_set_args(L, "i", ((int) names.size()) + 1);
 }
 
-ADE_FUNC(setPostEffect, l_Graphics, "string name, [number value=0]", "Sets the intensity of the specified post processing effect", "boolean", "true when successful, false otherwise")
+ADE_FUNC(setPostEffect, l_Graphics, "string name, [number value=0] [red 0.0 - 1.0] [green 0.0 - 1.0] [blue 0.0 - 1.0]", "Sets the intensity of the specified post processing effect. Optionally set RGB values for post effects that use them", "boolean", "true when successful, false otherwise")
```

Comment:
The parameter documentation isn't consistent with the rest of the code. Parameters should be separated by commas and for optional parameters the default value should be specified. The acceptable range should be specified in the function documentation and not in the parameter list.